0chain_dev_deployment.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
Will this script run on single terminal? How would I ensuer that all 3 miners or 2 sharders are up? How can I see their console logs & outputs if I setup chain using this script?

Reply:
`docker logs -f <container_name>` to check logs 
`docker ps` to check if all containers are up (we can update the script to do that as well good idea I guess 🤔  )
This does not clutter up the console as well as allows us to check the logs we care about